docs/proposals/edgemesh-design.md

```diff
@@ -76,6 +77,149 @@ Add two endpoints definition:
 |ServiceBus|Target|Request Forward|
 |Router|Source|Resutst Forward, Service Discovery, Load Balance|
 
+<img src="../images/proposals/router-design.png">
+
+* R1: ServiceController(SC) watches on k8s apiserver for Services/Endpoints.
+* R2: SC get the service definition and create router rule definition and save in DB(MySQL).
```

Comment:
Any specific reason that we choose MySQL?

I feel the router design is very alike kube-dns before 1.3(with skydns deployed). In that version of kube-dns, Kubernetes make use of etcd as etcd is very good at storing/watching metadata.

Reply:
Agreed. The only thought of using MySQL/sqlite was, since we don't have ETCD in edge and we have a design in proposal for EdgeSite, which uses k3s server which also replaced etcd with sqlite, I used SQL based DB to keep it compatable in both scenarios. Should I go-ahead and replace with ETCD ?
Thoughts @m1093782566 @qizha ??

Reply:
cc @CindyXing your thoughts ??

Reply:
I thnk it is OK to use ETCD on cloud

Reply:
Done, Thanks